src/redux/missions/missionsSlice.js

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable no-param-reassign */
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import axios from 'axios';
```

Comment:
Axios seems to not be installed as a dependency on package.json, could you install it using `npm install`

This causes that when I try to run the app it crashes because Axios is not on package.json